### PR TITLE
Ref #30962 -- Added GitHub action for static code linters

### PR DESCRIPTION
**Pull request by codingjoe**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -15,3 +15,4 @@ graft scripts
 graft tests
 global-exclude __pycache__
 global-exclude *.py[co]
+prune .github
```
Comment (carltongibson) [inline MANIFEST.in]: Any `MANIFEST` changes need a separate PR, and explanation/justification. (It's just too hot to pull in otherwise.)

**Comment by carltongibson** [comment]
Hey Joe, thanks for this. Interesting. 

Can I ask you to collaborate with @orf (and everyone else) over at https://github.com/orf/django-github-actions on this? 

(I saw your comment to the effect that we maybe don't need a long-running fork, but I'd be grateful if you could work that out with Tom — from my POV, it looks like it creates a space to experiment without creating a lot of noise for Mariusz and me on the main repo here, whilst it's all still Proof-of-Concept stage.) 

I'm going to close this now. It may be just fine, but we're not yet at the stage where the plan is agreed. 

Thanks! I'm quite optimistic about this topic. 💃

**Comment by codingjoe** [comment]
@carltongibson, I get the notion and I don't want to cause you any disturbance. But I'd like to ensure that all discussions are being archived. A for that lives only to discuss the current topic is not likely to survive past it's integration. I also don't think it's awefully intuitive for other who want to get involved. What do you think?

**Comment by carltongibson** [comment]
Errr...? I think Tom's work is looking good and that all efforts combined there is the quickest and best way for us to make progress as a group.

**Comment by codingjoe** [comment]
@carltongibson pardon my English my phone was about to die. I see your point, then again. I don't see the combined effort in @orf repo, it's only a couple commit ahead of Django HEAD all authored by @orf.

**Comment by carltongibson** [comment]
Hey @codingjoe. No problem. Your input is much appreciated at ever. 

I think the think here is that Tom (and Florian and ...) are already thinking about this, so talking with them is just the best way to collaborate. I'm pretty certain there is a lot to do to get this right. 🙂

(I'd imagine that as soon as there's agreement on how it should go we will have a PR here.)

**Comment by codingjoe** [comment]
@carltongibson I guess let's just stick to the mailing list for now.

**Pull request closed without merging**